tests/posix_semaphore/main.c

```diff
@@ -260,13 +273,15 @@ void test4(void)
             puts("first: timed out");
         }
     }
-    stop = xtimer_now_usec64() - start;
-    if ((stop < (exp - 100)) || (stop > (exp + 100))) {
-        fmt_u64_dec(uint64_str, stop);
+
+    uint64_str[fmt_u64_dec(uint64_str, elapsed)] = '\0';
+    if (elapsed < (exp - 100)) {
         printf("first: waited only %s usec => FAILED\n", uint64_str);
     }
+    else if (elapsed > (exp + TEST4_TIMEOUT_EXCEEDED_MARGIN))
```

Comment:
`{` missing

Reply:
My bad, I forgot to rerun 'static-test'.